inc/content-front.php

```diff
@@ -15,27 +15,46 @@
 	<div class="entry-content">
 		
 				<?php
+				// Get array of sticky posts.
 				$sticky = get_option( 'sticky_posts' );
-				rsort( $sticky );
-				$sticky = array_slice( $sticky, 0, 5 );
-				$the_query = new WP_Query( array( 'post__in' => $sticky, 'ignore_sticky_posts' => 1 ) );
-				while ( $the_query->have_posts() ) {
-					$the_query->the_post();
-				?>
-				
-				<div class="excerpt-post">
-					<img class="excerpt-post__fig"  src="<?php echo get_first_post_image(); ?>" width="200" >
-					<div class="excerpt-post__body">
-		                <h3><a href="<?php echo the_permalink();?>"><?php echo get_the_title();?></a></h3>
-		                <?php custom_excerpt( 20, '...' ) ?>
-					</div>
-				</div>
-				
-				<?php } wp_reset_postdata(); ?>
 
-				<?php the_content(); ?>
-				
-				<?php if ( is_active_sidebar( 'sidebar-two' ) ) : ?>
+				// If no sticky posts, skip this whole section.
+				if ( count( $sticky ) > 0 ) {
+
+					// Build WP query.
+					$args = array(
+						'posts_per_page' => 5,
+						'post__in' => $sticky,
+						'ignore_sticky_posts' => 1,
+					);
+
+					// Execute query.
+					$the_query = new WP_Query( $args );
+
+					if ( $the_query->have_posts() ) {
+
+						while ( $the_query->have_posts() ) :
+							$the_query->the_post();
+						?>
+
+						<div class="excerpt-post">
+							<img class="excerpt-post__fig" src="<?php echo esc_attr( get_first_post_image() ); ?>" width="200" >
+							<div class="excerpt-post__body">
+				                <h3><a href="<?php echo the_permalink();?>"><?php echo get_the_title();?></a></h3>
+				                <?php custom_excerpt( 20, '...' ) ?>
+							</div>
+						</div>
+
+						<?php endwhile;
+
+						wp_reset_postdata();
+
+					} // End "have posts".
+				} // End "if no sticky posts".
+
+				the_content();
+
+				if ( is_active_sidebar( 'sidebar-two' ) ) : ?>
```

Comment:
Line 57 isn't new, and is part of an existing check for the sidebar to make sure it is rendered only when needed. The original construct was

```
				<?php if ( is_active_sidebar( 'sidebar-two' ) ) : ?>
				
					<?php dynamic_sidebar( 'sidebar-two' ); ?>
					
				<?php endif; ?>		
```

Looking at this, I actually didn't go quite far enough - it is somewhat illegible IMO to be closing and opening PHP tags unnecessarily - so what might be better is:

```
				if ( is_active_sidebar( 'sidebar-two' ) ) :
				
					dynamic_sidebar( 'sidebar-two' );
					
				endif;

```